File: Tests/src/tests/BaseTest.java
package tests;

import java.io.IOException;
import java.security.KeyManagementException;
import java.security.KeyStoreException;
import java.security.NoSuchAlgorithmException;

import org.junit.jupiter.api.*;
import org.openqa.selenium.WebDriver;

import pages.*;
import data.Constants;
import utilities.Driver;
import data.Users;

public class BaseTest {
	private WebDriver driver;
	
	protected LoginPage loginPage;
	protected DepositPage depositPage;
	protected SettingsPage settingsPage;
	protected HistoryPage historyPage;
	
	@BeforeAll
	@AfterAll
	public static void beforeAll() throws IOException, NoSuchAlgorithmException, KeyStoreException, KeyManagementException {
		Users.deleteAll();
		Users.register(Constants.Login);
	}
	
	@BeforeEach
	public void initDriver() throws IOException {
		driver = Driver.initWebDriver();
		loginPage = new LoginPage(driver);
	}
	
	@AfterEach
    public void tearDown() {
        driver.quit();
    }
}

File: Tests/tests/BaseTest.py
import unittest
from selenium import webdriver
from selenium.webdriver.chrome.service import Service

from Constants import Constants
from utils import API


class BaseTestCase(unittest.TestCase):
    def setUp(self):
        API.Users.delete_all()
        API.Users.create(Constants.LOGIN)

        options = webdriver.ChromeOptions()
        options.add_argument('ignore-certificate-errors')

        # service = Service("./../chromedriver.exe")
        # self.driver = webdriver.Chrome(service=service, profile=profile)
        service = Service("./../chromedriver")
        self.driver = webdriver.Chrome(service=service, chrome_options=options)

    def tearDown(self) -> None:
        self.driver.quit()
